program/find-perimeter-of-a-square/FindPerimeterOfASquare.kt

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner
+
+fun main() {
+    print("Enter the side of the square: : ")
+    println()
+    var side = readLine()
+
+    try {
+        println("Perimeter of square with side $side : " + FindPerimeterOfASquare(side!!.toDouble()))
+    } catch (exception: Exception) {
+        println("Input string was not in a correct format")
+    }
```

Comment:
Code is throwing error irrespective of valid input

Reply:
works fine :)
![Screenshot from 2023-01-06 15-29-40](https://user-images.githubusercontent.com/54644599/210977594-7e35e37d-c9c7-453c-88c7-63f624154f9f.png)